pyinterprod/interpro/match.py

```diff
@@ -503,12 +478,22 @@ def update_database_site_matches(uri: str, databases: list):
 
         oracle.drop_table(cur, "INTERPRO.SITE_MATCH_NEW", purge=True)
 
-    for index in oracle.get_indexes(cur, "INTERPRO", "SITE_MATCH"):
+    cur.close()
+    con.close()
+
+    logger.info("complete")
+
+
+def check_index(uri, table):
```

Comment:
```suggestion
def rebuild_indexes(uri: str, table: str):
```

- More informative function name :monocle_face:
- Type hinting :label: